pyo3-macros/src/lib.rs

```diff
@@ -55,26 +67,152 @@ pub fn pyproto(_: TokenStream, input: TokenStream) -> TokenStream {
     .into()
 }
 
+/// A proc macro used to expose Rust structs as Python objects.
+///
+/// `#[pyclass]` accepts the following [parameters][2]:
+///
+/// |  Parameter  |  Description |
+/// | :-  | :- |
+/// | `name = "python_name"` | Sets the name that Python sees this class as. Defaults to the name of the Rust struct. |
+/// | `freelist = N`  |  Implements a [free list][10] of size N. This can improve performance for types that are often created and deleted in a row. Profile your code to see whether `freelist` is right for you.  |
```

Comment:
Slight wording adjustment.

```suggestion
/// | `freelist = N`  |  Implements a [free list][10] of size N. This can improve performance for types that are often created and deleted in quick succession. Profile your code to see whether `freelist` is right for you.  |
```

Reply:
That part was copied from the guide 😶